sympy/concrete/expr_with_limits.py

```diff
@@ -90,7 +91,13 @@ def _process_limits(*symbols):
                 limits.append(Tuple(V))
             continue
         elif is_sequence(V, Tuple):
-            V = sympify(flatten(V))
```

Comment:
You can leave this here: it will flatten Range into 3 elements (start, stop, step).

Reply:
Ah! that modification wasn't needed. I could have used the `(i, start, stop, step)` and converted it to `(i, start, stop)` if step is 1. I will make the change. I am surprised how can I miss that. 😅